tests/integration/linux_bridge_test.py

```diff
@@ -406,6 +409,159 @@ def test_port_vlan_not_implemented(port0_up):
         pass
 
 
+@pytest.mark.xfail(
+    reason="https://nmstate.atlassian.net/browse/NMSTATE-230", strict=True
+)
+class TestVlanFiltering:
+    Access_tag = 300
```

Comment:
nit: CamelCase are reserved for class names.

Reply:
Done.